Modules/Filtering/FFT/src/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 set(ITKFFT_SRCS
 itkComplexToComplexFFTImageFilter.cxx
-itkVnlFFTImageFilterInitFactory.cxx)
+itkVnlFFTImageFilterInitFactory.cxx
+itkFFTWFFTImageFilterInitFactory.cxx)
```

Comment:
For best practice, FFTW sources should be included in the build only if FFTW (or CUFFTW) is explicitly requested. This line should be removed and the conditional block below should be revisited.